pkg/controller/kubelet-config/helpers.go

```diff
@@ -22,7 +22,33 @@ import (
 	mcfgclientset "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned"
 )
 
-func createNewKubeletIgnition(jsonConfig []byte) ign3types.Config {
+func createNewKubeletLogLevelIgnition(level int32) *ign3types.File {
+	config := fmt.Sprintf(`
+[Service]
+Environment="KUBELET_LOG_LEVEL=%d"
+	`, level)
```

Comment:
This is creating some weird whitespace issues
```
sh-4.4# cat 20-logging.conf 

[Service]
Environment="KUBELET_LOG_LEVEL=3"
	sh-4.4#
```

Reply:
actually, this is more than a nit because if I go from having no `KubeletConfig` to having one that specifies log-level 2 (the default), I get an unexpected rollout because the default `20-logging.conf` doesn't match the rendered one.

Reply:
Given that this is a be probably should just fix it now instead of creating another bz to fix later.

Reply:
fixed!